### PR TITLE
Impact Analysis - CMS133, CMS157, CMS149, and CMS159

### PR DESCRIPTION
**Pull request by soy-park**
Impact analysis of QICore6 uplift for: 

-  **CMS133** (_Cataracts: 20/40 or Better Visual Acuity within 90 Days Following Cataract Surgery_)
- **CMS157** (_Oncology: Medical and Radiation – Pain Intensity Quantified_)
- **CMS149** (_Dementia: Cognitive Assessment_)
- **CMS159** (_Depression Remission at Twelve Months_)

For each of the above measures, I added the following comments to:
- Update 'Condition' to `ConditionProblemsHealthConcerns` and/or `ConditionEncounterDiagnosis`
- Update 'Observation' to `ObservationClinicalResult` or `ObservationScreeningAssessment`
- Update 'ObservationNotDone' to `ObservationCancelled`

### CODE REVIEW

File: input/cql/Cataracts2040BCVAwithin90DaysFHIR.cql [hunk 1]
```diff
@@ -86,6 +90,13 @@ define "Cataract Surgery Between January and September of Measurement Period":
       and CataractSurgery.performed.toInterval ( ) starts 92 days or more before end of "Measurement Period"
       and CataractSurgery.status = 'completed'
 
+/*
+@QICore6Uplift: 
+  For Conditions of "Burn Confined to Eye and Adnexa", "Choroidal Detachment", "Choroidal Hemorrhage and Rupture", "Cloudy Cornea", "Corneal Edema", "Central Corneal Ulcer", "Injury to Optic Nerve and Pathways",
+    "Open Wound of Eyeball", "Optic Neuritis", "Purulent Endophthalmitis", "Retinal Detachment with Retinal Defect", "Traumatic Cataract", and "Scleritis", 
+    * Update Condition to ConditionEncounterDiagnosis 
+  For the rest of the Conditions not listed above, update Condition to ConditionProblemsHealthConcerns
+*/
```
Comment (jbyrdevans) [inline input/cql/Cataracts2040BCVAwithin90DaysFHIR.cql:99]: Should possibly review with the steward for clinical intent, but as the logic stands this should be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

File: input/cql/DementiaCognitiveAssessmentFHIR.cql [hunk 2]
```diff
@@ -52,6 +59,9 @@ define "Assessment of Cognition Using Standardized Tools or Alternate Methods":
     where CognitiveAssessment.value is not null
       and CognitiveAssessment.status in { 'final', 'amended', 'corrected', 'preliminary' }
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/
```
Comment (jbyrdevans) [inline input/cql/DementiaCognitiveAssessmentFHIR.cql:64]: This should be the union of ConditionEncounterDiagnosis and ConditionProblemsHealthConcerns

File: input/cql/DepressionRemissionatTwelveMonthsFHIR.cql [hunk 3]
```diff
@@ -71,11 +78,17 @@ define "SDE Sex":
...
+@QICore6Uplift: Update 'Observation' to 'ObservationScreeningAssessment'
+*/ 
 define "Depression Assessments Greater than 9":
   [Observation: "PHQ 9 and PHQ 9M Tools"] DepressionAssessment
     where DepressionAssessment.value > 9
       and DepressionAssessment.status in { 'final', 'amended', 'corrected' }
 
+/*
+@QICore6Uplift: Update both 'Condition' resources to 'ConditionProblemsHealthConcerns'
+*/
```
Comment (jbyrdevans) [inline input/cql/DepressionRemissionatTwelveMonthsFHIR.cql:91]: Should confirm clinical intent, but as written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

File: input/cql/OncologyPainIntensityQuantifiedFHIR.cql [hunk 4]
```diff
@@ -38,6 +42,9 @@ define "Chemotherapy Within 31 Days Prior and During Measurement Period":
     where ChemoAdministration.performed.toInterval ( ) during Interval[start of "Measurement Period" - 31 days, end of "Measurement Period"]
       and ChemoAdministration.status = 'completed'
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/
```
Comment (jbyrdevans) [inline input/cql/OncologyPainIntensityQuantifiedFHIR.cql:47]: As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

File: input/cql/OncologyPainIntensityQuantifiedFHIR.cql [hunk 5]
```diff
@@ -58,6 +65,9 @@ define "Numerator 1":
     with "Standard Pain Assessment with Result" PainAssessed
       such that PainAssessed.effective.toInterval ( ) during FaceToFaceOrTelehealthEncounterWithChemo.period
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/
```
Comment (jbyrdevans) [inline input/cql/OncologyPainIntensityQuantifiedFHIR.cql:70]: as written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

**Review by jbyrdevans — CHANGES_REQUESTED** [review]

File: input/cql/Cataracts2040BCVAwithin90DaysFHIR.cql (hunk 1 above)
Comment (soy-park) [reply input/cql/Cataracts2040BCVAwithin90DaysFHIR.cql:99]: So, for the following example, would it be option 1 or option 2? 

Option 1: 
```
with ([ConditionEncounterDiagnosis: "Acute and Subacute Iridocyclitis"]
   union [ConditionProblemsHealthConcerns: "Amblyopia"]
```

Option 2: 
```
with ([ConditionEncounterDiagnosis: "Acute and Subacute Iridocyclitis"]
   union [ConditionProblemsHealthConcerns: "Acute and Subacute Iridocyclitis"]
   union [ConditionEncounterDiagnosis: "Amblyopia"]
   union [ConditionProblemsHealthConcerns: "Amblyopia"]
```

**Review by jbyrdevans — APPROVED** [review]

**Pull request merged**